shiny/examples/include_css/css/styles.css

```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: 'Square Peg';
+  src: url('SquarePeg-Regular.ttf');
```

Comment:
I think that this example shouldn't include the font, because it lives in `/shiny/examples`. That means that the font will be distributed with the wheel file (and make it 120KB bigger), but almost no one will ever make use of it.

We could put an example with a font in `/examples`, because that dir isn't distributed with the built package.

Reply:
This was marked as resolved, but I think the line of code still needs to be removed -- otherwise, won't the CSS still try to load the font?

Reply:
Whoops, you're correct, sorry about missing that.